Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -52,6 +53,7 @@ static __pthread_mutex_t s_loader_lock = __PTHREAD_MUTEX_INITIALIZER;
 
 static bool s_allowed_to_check_environment_variables { false };
 static bool s_do_breakpoint_trap_before_entry { false };
+static char* s_ld_library_path { nullptr };
```

Comment:
...Immediately after that last one:
```suggestion
static StringView s_ld_library_path { };
```